### DISCUSSION TITLE
Route Redirect

### DISCUSSION THREAD

**Discussion by GlenWarren** [orchidsoftware/platform#2528] [Q&A]
Is it possible to redirect a route path to another one?

Currently, if someone goes to path `/home` they get a 404 Not Found error. When a user is already logged in, if they navigate to `/login`, they are automatically redirected to `/home` which results in 404.

I have `'home' => '/admin/main',` set in my fortify config.

How can I redirect users to `'/admin/main'` if they navigate to `/home`? Can I add custom redirects in the `routes.php` file?

I have also previously been unable to fix the redirect when a user logs out. See here: https://github.com/orchidsoftware/platform/discussions/2206

**Comment by tabuna** [comment]
This packet does not use the `home` route. You may have it left over from previous versions of Laravel. 
You can use https://laravel.com/docs/9.x/routing#redirect-routes to redirect it. (In routes/web.php)

**Reply by GlenWarren** [reply]
Thanks. For some reason using `Route::redirect('/home', '/admin/main')` did not work.

**Reply by tabuna** [reply]
Perhaps it will be enough for you to change this https://github.com/laravel/laravel/blob/5b60b604c4167b30286a6e033723a6b274e8452d/app/Providers/RouteServiceProvider.php#L20

**Reply by gioacchinopoletto** [reply]
> Perhaps it will be enough for you to change this https://github.com/laravel/laravel/blob/5b60b604c4167b30286a6e033723a6b274e8452d/app/Providers/RouteServiceProvider.php#L20

Hi @tabuna I have try to change it to `/admin/main` but not work... (my goal is not have a frontend for now)

**Reply by tabuna** [reply]
Then for help, I need more details so that I can reproduce the problem.

**Reply by gioacchinopoletto** [reply]
> Then for help, I need more details so that I can reproduce the problem.

Temporary I have solved my problem on `/routes/web` adding:

```
use Orchid\Platform\Http\Controllers\IndexController;

/* Laravel HOME costant: redirect to Orchid dashboard */
Route::get("/home", [IndexController::class, "index"])->name("orchid.home");
```
